client/src/pages/WhatIsAutism.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import '../../App.css';
+
+export default function WhatIsAutism() {
+  return <h1 className='sign-up'>LIKE & SUBSCRIBE</h1>;
```

Comment:
page name is not representing the page content, is this just an first element of the page and the page will be updated?